packages/headless-styles/src/components/ErrorMessage/errorMessageCSS.ts

```diff
@@ -4,7 +4,7 @@ import {
   createErrorMessageProps,
 } from './shared'
 import type { ErrorMessageOptions } from './types'
-import styles from './errorMessageCSS.module.css'
+import './errorMessageCSS.scss'
 
 const ERROR_MESSAGE = 'ps-error-message'
```

Comment:
Please help out here and also convert old global selectors to use `pando` key.

```suggestion
const ERROR_MESSAGE = 'pando-error-message'
```